docs/installation/deployment/kubernetes.rst

```diff
@@ -204,194 +74,82 @@ You can either clone the https://github.com/open-zaak/open-zaak repository,
 
-The ``apps.yml`` playbook sets up the Open Zaak and Open Notificaties
-installations.
+* installs Open Zaak
+* installs Open Notificaties
 
-I already have an ingress-controller
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can run the Ansible-playbooks as-is (with some configuration through variables), or
+use them as inspiration for manual deployment.
 
-.. todo:: opt-out of the Traefik CRD and provide an alternative Ingress resource
+For a list of all the available variables, check the
+`collection <https://github.com/open-zaak/ansible-collection>`_ roles.
 
-Configuring Open Zaak
-^^^^^^^^^^^^^^^^^^^^^
+**Configuring Open Zaak**
 
 To deploy Open Zaak, some variables need to be set (in ``vars/open-zaak.yml``):
 
 * ``openzaak_domain``: the domain name, e.g. ``open-zaak.gemeente.nl``
 * ``openzaak_secret_key``: generate a key via https://miniwebtool.com/django-secret-key-generator/.
   Make sure to put the value between single quotes!
 
-See ``roles/openzaak/defaults/main.yml`` for other possible variables to
-override. You might want to tweak environment variables in order to
+You might want to tweak environment variables in order to
 :ref:`provision a superuser<installation_provision_superuser>`.
 
-Configuring Open Notificaties
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**Configuring Open Notificaties**
 
 To deploy Open Notificaties, some variables need to be set (in ``vars/open-notificaties.yml``):
 
 * ``opennotificaties_domain``: the domain name, e.g. ``open-notificaties.gemeente.nl``
 * ``opennotificaties_secret_key``: generate a key via https://miniwebtool.com/django-secret-key-generator/.
   Make sure to put the value between single quotes!
 
-See ``roles/opennotificaties/defaults/main.yml`` for other possible variables to
-override.
-
-Configuring NLX
-^^^^^^^^^^^^^^^
-
-If you want to use NLX, you need to create NLX certificate. See https://docs.nlx.io/try-nlx/docker/retrieve-a-demo-certificate.
-The key and certificate need to be set in ``vars/nlx.yml``:
-
-* ``nlx_inway_key``: private key
-* ``nlx_inway_cert``: certificate
-
-
-Deploying the applications
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Make sure your ``kubectl`` command can access the correct cluster:
-
-.. code-block:: shell
-
-    (env) [user@host]$ kubectl cluster-info
-
-Run the ``apps.yml`` playbook using:
+Deployment tooling - Helm
+=========================
 
-.. code-block:: shell
+If you're familiar with or prefer Helm_, there are community-provided Helm charts_
+for Open Zaak and Open Notificaties.
 
-    (env) [user@host]$ ./deploy.sh apps.yml
+.. note::
 
-.. _Kubernetes: https://kubernetes.io/
-.. _NGINX: https://www.nginx.com/
-.. _Ansible: https://www.ansible.com/
+   These charts are contributed by the community on a best-effort basis. The Technical
+   Steering Group takes no responsibility for the quality or up-to-date being of these
+   charts. This may lead to
+   `scary reports <https://github.com/open-zaak/open-zaak/issues/1097>`_ that are
+   easily misinterpreted.
```

Comment:
> This may lead to
   `scary reports <https://github.com/open-zaak/open-zaak/issues/1097>`_ that are
   easily misinterpreted.

I would leave this sentence out as I think it is rather confusing and not directly related to the Helm charts.

Reply:
wrong target audience indeed, removed.